.github/workflows/wheels.yml

```diff
@@ -41,13 +41,24 @@ jobs:
     - name: Build macos wheels
       if: startsWith(matrix.os, 'macos')
       run: |
+        python -m pip install --upgrade pip
         pip install -r requirements.txt
         python setup.py bdist_wheel
+        pip install delocate
+        cd dist
+        delocate-wheel -v `ls`
```

Comment:
Probably not relevant for the GitHub actions, but when I try to run this in my local machine it fails with the following message:
```
FileNotFoundError: [Errno 2] No such file or directory: 'otool'
```

Reply:
Yes, this is a macos specific tool, on linux we have `ldd` which does the same job.